tests/test_store.py

```diff
@@ -31,6 +32,11 @@
 logger = logging.getLogger(__name__)
 
 
+@contextmanager
+def does_not_raise():
```

Comment:
Could we place this out into a utility class so it doesn't need to be copied over to everywhere it's used?


I wonder if pytest might also have something like this that we can use without needing to write it ourselves.

Reply:
@BryanFauble Yeah, would it be appropriate to move to the `conftest.py`?

Reply:
@BryanFauble yeah that makes sense